### PR TITLE
docs: vueschool free-weekend banner

### PR DESCRIPTION
**Pull request by rahaug**
This PR adds a banner on top of router.vuejs.org to inform users about Vue School’s free-weekend promotion.

The banner is closable and opens Vue School's free-weekend promo page in a new tab. The banner will not reappear if a user has closed it (stored in local storage).

The banner is based on the one used on vuejs.org.

[Click here to see screenshots of the banner](https://imgur.com/a/z3f1g4M)

### CODE REVIEW

File: docs/.vuepress/theme/components/BannerTop.vue [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  </a>
+</template>
+
+<style lang="stylus">
+$topBannerHeight ?= 5rem
+$topBannerHeightMobile ?= 3.125rem
+$navbarHeight ?= 3.6rem
+$contentClass = '.theme-default-content'
+
+@import url('http://fonts.googleapis.com/css?family=Roboto')
```
Comment (posva) [inline docs/.vuepress/theme/components/BannerTop.vue]:
```suggestion
@import url('https://fonts.googleapis.com/css?family=Roboto')
```

File: docs/.vuepress/theme/components/BannerTop.vue [hunk 2]
```diff
@@ -0,0 +1,180 @@
+<template>
+  <a id="vs" href="https://vueschool.io/free-weekend?friend=vuerouter" target="_blank" rel="noreferrer">
+    <div class="vs-iso">
+      <img src="/images/vueschool/vs-iso.svg">
```
Comment (posva) [inline docs/.vuepress/theme/components/BannerTop.vue]:
```suggestion
      <img src="/images/vueschool/vs-iso.svg" alt="Vue School Logo">
```

File: docs/.vuepress/theme/components/BannerTop.vue [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    </div>
+    <div class="vs-logo">
+      <img src="/images/vueschool/vs-logo.svg">
```
Comment (posva) [inline docs/.vuepress/theme/components/BannerTop.vue]:
```suggestion
      <img src="/images/vueschool/vs-logo.svg" alt="Vue School Logo">
```

**Review by posva — CHANGES_REQUESTED** [review]
I think these changes make sense and should probably be ported to vue next too

**Comment by rahaug** [comment]
Good catch @posva, thanks for the suggestions! I've applied your suggestions.

I'll look into the docs-next banner.

**Review by posva — APPROVED** [review]

**Pull request merged**